source/app/index.js

```diff
@@ -182,29 +198,65 @@ function saveCurrent() {
 function setMenu() {
     const menuTemplate = [{
         label: "File",
-        submenu: [{
+        submenu: [
+            {
                 label: "New",
-                click: () => createWindow()
+                click: () => createWindow(),
+                accelerator: "CmdOrCtrl+N"
             },
             {
                 label: "Open",
-                click: openInCurrent
+                click: openInCurrent,
+                accelerator: "CmdOrCtrl+O"
             },
             {
                 label: "Save",
-                click: saveCurrent
+                click: saveCurrent,
+                accelerator: "CmdOrCtrl+S"
             }
         ]
+    }, {
+        label: 'Edit',
+        submenu: [
+            { role: 'undo' },
+            { role: 'redo' },
+            { type: 'separator' },
+            { role: 'cut' },
+            { role: 'copy' },
+            { role: 'paste' },
+            { role: 'pasteandmatchstyle' },
+            { role: 'delete' },
+            { role: 'selectall' }
+        ]
+    }, {
+        label: 'View',
+        submenu: [
+            { role: 'reload' },
```

Comment:
@sallar I think this might break.. There's some communication that happens with the window when loading.. not sure if this is exactly needed.